modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/build_fragment.ts

```diff
@@ -12,12 +12,12 @@
  * details.
  */
 
-export default function buildFragment(htmlString) {
+export default function buildFragment(htmlString: string) {
 	const div = document.createElement('div');
 
 	div.innerHTML = `<br>${htmlString}`;
 
-	div.removeChild(div.firstChild);
+	div.removeChild(div.firstChild!);
```

Comment:
Not sure if there's a better way... plaguing our codebase with non-null assertions doesn't feel right, yet it's certainly a powerful tool 😉

Reply:
Yeah, I agree, as it is explicit above that we have a children I think this makes it safe